### DISCUSSION TITLE
Overmask Slider is not working

### DISCUSSION THREAD

**Discussion by egormly** [zero01101/openOutpaint#88] [Q&A]
I have tried everything and I am unsure of what I might be doing.  Everything else seems to work, and the same type of slider for cursor size works, I click on it and am able to slide to different sizes, but for the overmask slider, I get nothing.  It will not move or let me edit in any way.  All the sliders in other features work (mask Brush, Img2Img).  I have the lastest Auto as of right now via git pull.

**Comment by seijihariki** [comment]
Does it simply not move?

**Reply by egormly** [reply]
It will not do anything, like it's not a slider box like the others. No interaction at all.

**Reply by seijihariki** [reply]
If you can send a short video to us here, I think it would be easier for us to diagnose. Any console messages in devtools?

**Reply by egormly** [reply]
I guess this can be deleted/marked solved, I rebooted, restarted everything and now it's working?  Very odd. I have to assume this was a browser issue maybe?

**Reply by seijihariki** [reply]
Very... strange. Probably some cache issue then?

**Reply by zero01101** [reply]
yeah, i've never experienced _anything_ like that... super weird :/

**Reply by egormly** [reply]
Not sure, using Edge maybe that was it. (lol)

**Comment by zero01101** [comment]
possible fix/incompatibility discovered: microsoft editor browser extension!?!? 

![image](https://user-images.githubusercontent.com/1649724/209564875-da53c119-f243-4bdd-b808-ef6df977a4ef.png)

**Reply by seijihariki** [reply]
That may be the case, what exactly does this extension do?

**Reply by zero01101** [reply]
lmao [spelling and grammar checker](https://chrome.google.com/webstore/detail/microsoft-editor-spelling/gpaiobkfhnonedkhhfjpmhdalgeoebfa?hl=en) which apparently gets offended enough at "misspelled" words like overmask that it outright says NO YOU CAN'T TOUCH THIS CLEARLY BROKEN THING
